supergraph.yaml

```diff
@@ -1,3 +1,4 @@
+federation_version: 1
```

Comment:
would prefer to leave `federation_version` out to ensure everything continues working fine for existing workflows that are using older gateways and that we're defaulting to `federation_version: 1` if it's omitted.